pkg/occlient/occlient.go

```diff
@@ -2076,6 +2076,63 @@ type Service struct {
 	PlanList []string
 }
 
+// GetServiceClass retrieves the ClusterServiceClass identified by the specified external name
+func (c *Client) GetServiceClass(externalName string) (class *scv1beta1.ClusterServiceClass, err error) {
+	classes, err := c.serviceCatalogClient.ClusterServiceClasses().List(metav1.ListOptions{
+		FieldSelector: "spec.externalName==" + externalName,
+	})
+
+	if len(classes.Items) != 1 {
+		return nil, nil
```

Comment:
bit confused, if != 1, we return no items and no error message? Should we error out if there is no ClusterServiceClass?

Reply:
Indeed.

Reply:
Fixed.